workflows/utilities/wf_read_QC_trim_pe.wdl

```diff
@@ -107,6 +109,17 @@ workflow read_QC_trim_pe {
         midas_db = midas_db
     }
   }
+  if ("~{workflow_series}" == "theiaprok") {
```

Comment:
Nice! I like how this is done by taking advantage of our workflow series tag

Reply:
It was a suggestion by @michellescribner ! :D